modules/configuring-log-visualizer.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * logging/log_visualization/log-visualization.adoc
+// * logging/cluster-logging-deploying.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-log-visualizer_{context}"]
+= Configuring the log visualizer
+
+You can configure which log visualizer type your {logging} uses by modifying the `ClusterLogging` custom resource (CR).
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have installed the {oc-first}.
+* You have installed the {clo}.
++
+[IMPORTANT]
+====
+If you want to use the {product-title} web console for visualization, you must enable the Console plugin during the Operator installation.
```

Comment:
I think this checkbox may just add a label or other thing to a resource.  @jgbernalp do you recall?  If so, we can expand by stating: "check the box or do..."

Reply:
Would this be covered by https://access.redhat.com/solutions/7036927 (https://issues.redhat.com/browse/OBSDOCS-555)? If so, I would leave the note for now, and I can revisit this when I'm working on that Jira for the console docs, since there's follow up to do here anyway?